src/main/java/com/greenfoxacademy/moviedatabaseapi/security/jwt/JwtUtil.java

```diff
@@ -0,0 +1,72 @@
+package com.greenfoxacademy.moviedatabaseapi.security.jwt;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+@Service
```

Comment:
not a service actually, you can use it in other classes statically:
```
JwtUtil.createToken(...)
```
but you need to change all the methods of this class to be public ones